docs/using.rst

```diff
@@ -722,7 +722,28 @@ Example usage for DNS-01 (Cloudflare API v4) (for example purposes only, do not
        fi
    fi
 
+.. _lock-files:
 
+Lock Files
+==========
+
+When processing a validation Certbot writes a number of lock files on your system
+to prevent multiple instances from overwritting each other's changes. This means
+that two instances of Certbot will not be able to run in parallel.
+
+Since there's no one single cross-platform place for these lock files, Certbot
+will write a lock file for all of the directories it uses. This include Certbot's
```

Comment:
Rather than framing this as a limitation of Certbot or the UNIX world, I'd say something like "Since the directories used by Certbot are configurable, Certbot will write..."

For more info, you can read about why we made this decision at the bottom of [this post](https://github.com/certbot/certbot/pull/4449#issuecomment-292578050). Feel free to leave this sentence as is if you like it better now.